target/linux/realtek/files-5.10/include/dt-bindings/clock/rtl83xx-clk.h

```diff
@@ -0,0 +1,15 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
```

Comment:
While it pains me to say; and I much much more prefer this style as well; I think upstream prefers
```suggestion
// SPDX-License-Identifier: GPL-2.0-only
```
Also, its fine to mark it as `GPL-2.0-or-later`. YOUR choice as author, I personally prefer to allow an upgrade (if needed,r atter then blocking it potentially forever)

Reply:
The (docs)[https://www.kernel.org/doc/html/v5.19/process/license-rules.html] say Markus used the correct style for C headers.

Reply:
Ah they made it even more confusing, why I still don't get. So `/*` for headers, `//` for C files. I see the reason for the change (good, I'm happy :p) what I don't get, why not make it consistent and just use `/*` for both. There's no _real_ C++ style comments in the kernel base, except for these SPDX identifiers, and then they're not consistent. Someone must have felt really strongly about wanting C++ headers and probably hoped this was 'a way in'. Especially considering the reason for the rollback on headers, might as well have done them both for consistency sake.

Anyway, sorry for the noise :)